ui/raidboss/data/04-sb/raid/o6n.ts

```diff
@@ -24,19 +37,36 @@ const triggerSet: TriggerSet<Data> = {
       response: Responses.tankBuster(),
     },
     {
-      id: 'O6N Meteors',
+      id: 'O6N Demonic Pain',
+      type: 'Tether',
+      // 0001 = far enough, 0039 = too close
+      netRegex: NetRegexes.tether({ id: ['0001', '0039'] }),
+      condition: (data, matches) => data.me === matches.target || data.me === matches.source,
+      suppressSeconds: 10,
+      alertText: (data, matches, output) => {
+        const partner = data.me === matches.target ? matches.source : matches.target;
+        return output.text!({ player: data.ShortName(partner) });
```

Comment:
Isn't this always a tether between the player and the boss on normal mode?

Reply:
Thank you for catching this!!